c3pzero_base_config/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>c3pzero_base_config</name>
+  <version>0.1.0</version>
+  <description>base config for c3pzero</description>
+  <maintainer email="abake48@gmail.com">Anthony Baker</maintainer>
+  <license>Unlicense</license>
```

Comment:
Better to set this to Proprietary IMO, to be safe RE: how we'll use this code (unless we've already set a pattern elsewhere in this project).

```suggestion
  <license>Proprietary</license>
```